### PR TITLE
Fix testing documentation for integration tests

### PR DESCRIPTION
**Pull request by davideicardi**
It was not clear how to run integration tests, I have fixed and improved the example.

### CODE REVIEW

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md [hunk 1]
```diff
@@ -286,10 +286,16 @@ The standard source hierarchy is used:
     test classpath
 
 The standard testing tasks are available, but must be prefixed with
-`it:`. For example,
+`it:`. For example to run all integration tests:
```
Comment (eed3si9n) [inline src/reference/02-DetailTopics/02-Configuration/14-Testing.md]:
```suggestion
`IntegrationTest/`. For example to run all integration tests:
```

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md [hunk 2]
```diff
@@ -286,10 +286,16 @@ The standard source hierarchy is used:
...
 
 ```
-> IntegrationTest / testOnly org.example.AnIntegrationTest
+> it:test
```
Comment (eed3si9n) [inline src/reference/02-DetailTopics/02-Configuration/14-Testing.md]:
```suggestion
> IntegrationTest/test
```

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md [hunk 3]
```diff
@@ -286,10 +286,16 @@ The standard source hierarchy is used:
...
+```
+
+Or to run a specific test:
+
+```
+> it:testOnly org.example.AnIntegrationTest
```
Comment (eed3si9n) [inline src/reference/02-DetailTopics/02-Configuration/14-Testing.md]:
```suggestion
> IntegrationTest/testOnly org.example.AnIntegrationTest
```

**Comment by eed3si9n** [comment]
@davideicardi Thanks. I think we should unify towards the slash syntax.

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md (hunk 1 above)
Comment (davideicardi) [reply src/reference/02-DetailTopics/02-Configuration/14-Testing.md]: Fixed

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md (hunk 2 above)
Comment (davideicardi) [reply src/reference/02-DetailTopics/02-Configuration/14-Testing.md]: Fixed

File: src/reference/02-DetailTopics/02-Configuration/14-Testing.md (hunk 3 above)
Comment (davideicardi) [reply src/reference/02-DetailTopics/02-Configuration/14-Testing.md]: Fixed

**Review by eed3si9n — APPROVED** [review]

**Pull request merged**